carrier_account_environment/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,30 @@
+With this module installed, the delivery carrier are
+configured in the `server_environment_files` module (which is a module
```

Comment:
can you also talk about the alternatives like [server_environment_data_encryption](https://github.com/OCA/server-env/tree/14.0/server_environment_data_encryption)